spirv_glsl.cpp

```diff
@@ -501,6 +501,11 @@ void CompilerGLSL::find_static_extensions()
 		switch (cap)
 		{
 		case CapabilityShaderNonUniformEXT:
+			if (!options.vulkan_semantics)
+				require_extension_internal("GL_NV_gpu_program5");
```

Comment:
Should probably apply this to RuntimeDescriptorArrayEXT as well?

Reply:
well not really, since open ended arrays of descriptors (or any bindings) are not legal in OpenGL GLSL AFAIK.

basically this lets me divergently index SIZED arrays of descriptors on Nvidia GPUs, nothing more really

Reply:
are we resolved?

Reply:
yes